io-engine-tests/src/fio.rs

```diff
@@ -0,0 +1,199 @@
+use super::file_io::BufferSize;
+use std::sync::atomic::{AtomicU32, Ordering};
+
+/// TODO
+#[derive(Debug, Clone)]
+#[allow(dead_code)]
+pub struct FioJob {
+    /// Job counter.
+    counter: u32,
+    /// Job name.
+    name: String,
+    /// I/O engine to use. Default: spdk.
+    ioengine: String,
+    /// Filename.
+    filename: String,
+    /// If true, use non-buffered I/O (usually O_DIRECT). Default: true.
+    direct: bool,
+    /// Block size for I/O units. Default: 4k.
+    blocksize: u32,
+    /// Number of I/O units to keep in flight against the file.
+    iodepth: u32,
+    /// Number of clones (processes/threads performing the same workload) of
+    /// this job. Default: 1.
+    numjobs: u32,
+    /// Terminate processing after the specified number of seconds.
+    runtime: u32,
+    /// Total size of I/O for this job.
+    size: BufferSize,
+}
+
+impl Default for FioJob {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl FioJob {
+    pub fn new() -> Self {
+        static JOB_COUNTER: AtomicU32 = AtomicU32::new(0);
+
+        let counter = JOB_COUNTER.fetch_add(1, Ordering::SeqCst);
+
+        Self {
+            counter,
+            name: format!("fio-{counter}"),
+            ioengine: "spdk".to_string(),
+            filename: String::new(),
+            direct: true,
+            blocksize: 4096,
```

Comment:
Here it's hardcoded 4K

Reply:
It's the same default as fio's. I'll make it optional to avoid confusion.